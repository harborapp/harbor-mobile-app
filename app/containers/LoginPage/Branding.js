import React from 'react'

import styles from './styles'
import messages from './messages'
import H1 from '../../components/H1'
import Container from '../../components/Container'
import FormattedMessage from '../../components/FormattedMessage'

/**
 * Branding renders the static title and subtitle
 * in the login pages.
 *
 * @returns {jsx}
 */
const Branding = () => (
  <Container centerChildren>
    <React.Fragment>
      <H1>
        <FormattedMessage
          {...messages.brandingTitle}
          style={styles.brandingTitleText}
        />
      </H1>
      <FormattedMessage
        {...messages.brandingSubtitle}
        style={styles.brandingSubtitleText}
      />
    </React.Fragment>
  </Container>
)

export default Branding
