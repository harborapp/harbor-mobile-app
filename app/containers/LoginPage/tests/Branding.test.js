/* eslint-env jest */
import React from 'react'
import { IntlProvider } from 'react-intl'
import renderer from 'react-test-renderer'

import Branding from '../Branding'

describe('Branding', () => {
  it('should render', () => {
    const tree = renderer.create(
      <IntlProvider locale='en'>
        <Branding />
      </IntlProvider>
    )

    expect(tree).toMatchSnapshot()
  })
})
