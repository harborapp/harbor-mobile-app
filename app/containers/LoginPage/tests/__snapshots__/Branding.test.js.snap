// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Branding should render 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
      Object {
        "alignItems": "center",
        "justifyContent": "center",
      },
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontSize": 64,
        },
        undefined,
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      description="Title for the App"
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#FFFFFF",
          "elevation": 1,
          "fontFamily": "NunitoSans-Bold",
        }
      }
    >
      harbor
    </Text>
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    description="Sub title for the App"
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#FFFFFF",
        "fontFamily": "NunitoSans-Regular",
        "fontSize": 14,
      }
    }
  >
    Painless release cycles for Mobile apps
  </Text>
</View>
`;
